assets/panel/debugger.properties

```diff
@@ -798,9 +798,9 @@ gotoLineModal.title=Go to a line number in a file
 
 # LOCALIZATION NOTE(gotoLineModal.key2): The shortcut for opening the
```

Comment:
I can't jump into this right away but note to self or @gregtatum ; I think we need to change `key2` to `key3` here.

Reply:
Thanks for catching that, fixed! For some reason my editor is not wanting to search through the assets folder.